src/app/components/navbar/NavBar.jsx

```diff
@@ -83,12 +82,20 @@ const NavBar = props => {
                                 Users and access
                             </Link>
                         </li>
-
-                        <li className="global-nav__item">
-                            <Link to={`${rootPath}/teams`} activeClassName="selected" className="global-nav__link">
-                                Teams
-                            </Link>
-                        </li>
+                        {props.config?.enableNewSignIn && (
+                            <li className="global-nav__item">
+                                <Link to={`${rootPath}/groups`} activeClassName="selected" className="global-nav__link">
+                                    Preview teams
```

Comment:
what is the difference between `Preview teams` and `Teams`? you made the routes to `/groups`. We need to make sure next dev come after you and understand what it this about I think using consistent naming would help

Reply:
'Team' is a legacy term users used to see
'Preview team' is the correct term going forward for displaying to the user

As there will only be one type of 'team' in Florence which is a preview 'team' I think it is safe to refer to 'preview team' as just 'team' in the code as we have been doing all along, the only real change is that users see a more descriptive name of what a 'team' is.

A Preview team is a collection of users who are not publishers nor admins (so Business area users rather than Publishing team users in DP). These users get added to a Preview Team and that Preview Team is added to collections within Florence and those collections contain content due to be published. This gives the Business areas the opportunity to read publications and review them before they get published.

The term itself was decided upon by our Content Designer after they consulted with the Publishing team. So going forward the correct full term for what was 'teams' is now 'Preview Teams' (original Florence I don't think had any content designer involved). (note it is actually technically possible to add Publishers and Admins to preview teams, but this grants them no extra access as they will already have access due to their role).

Hope that clears that up a little, in terms of 'groups' I have added a comment onto another thread you commented on in the api-clients.